Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -5,44 +5,60 @@
 import StreamChat
 import UIKit
 
-// TODO: Audit
-public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
-    public var title: String? {
-        get { titleLabel.text }
-        set { titleLabel.text = newValue }
+public typealias ChatMessageListTitleView = _ChatMessageListTitleView<NoExtraData>
+
+/// A view that is used as a wrapper for status data in navigationItem's titleView
+open class _ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
```

Comment:
Rather than `_ChatMessageListTitleView` can we just name it in a more general way? It has nothing specific for message list view, it's just a title view with two labels :) Maybe `TitleConainterView` or it's too much?

Reply:
let's name it `TitleContainerView`, it seems good to me 🙂